crates/aptos/src/common/init.rs

```diff
@@ -208,7 +217,7 @@ impl CliCommand<()> for InitTool {
         };
 
         // Public key
-        let public_key = if self.ledger {
+        let public_key = if self.ledger || self.hardware_wallet_options.is_hardware_wallet() {
```

Comment:
nit: global function for `if self.ledger || self.hardware_wallet_options.is_hardware_wallet()` ?

Reply:
Yeah, you can put this function on `self`. `fn is_hardware_wallet(&self) -> bool`